packages/tools/package/src/commands/changelog/changelog.utils.ts

```diff
@@ -0,0 +1,118 @@
+import { Package } from '../../models';
+import { AlongWithData, ChangelogEntry } from './changelog.types';
+
+const fs = require('fs');
+const path = require('path');
+const { exec } = require('child_process');
+
+/**
+ * Function to execute shell command and capture output
+ * @param scriptPath - Path to the script which we want to run.
+ * @returns - Promise that resolves once the script is run and returns the output.
+ */
+export function runShellScript(scriptPath: string) {
+  return new Promise((resolve, reject) => {
+    // @ts-ignore
+    exec(`bash ${scriptPath}`, (error, stdout, stderr) => {
+      if (error) {
+        console.error(`Error executing script: ${error}`);
+        reject(error);
+      } else if (stderr) {
+        console.error(`Script stderr: ${stderr}`);
+        reject(stderr);
+      } else {
+        // Process the stdout (output) as needed
+        resolve(stdout);
+      }
+    });
+  });
+}
+
+/**
+ * Function to get and formate alongWith object
+ * alongWith object has the information about packages that are modified `along with`
+ * a package
```

Comment:
Description could be a bit more structured

Reply:
Ive tried to make the description a bit more clear.